src/newViews/Tagging/Sources/LeadEditModal/LeadEditForm/schema.ts

```diff
@@ -0,0 +1,131 @@
+import {
+    ObjectSchema,
+    requiredStringCondition,
+    requiredCondition,
+    PartialForm,
+    urlCondition,
+} from '@togglecorp/toggle-form';
+
+import {
+    KeyValueElement,
+} from '#typings';
+
+export type LeadSourceType = 'text' | 'disk' | 'website' |
+    'dropbox' | 'google-drive' | 'rss-feed' | 'emm' | 'web-api' | 'unknown';
+
+export interface EmmEntityOption {
+    key: number;
+    label: string;
+    totalCount: number;
+}
+
+export interface EmmTrigger {
+    emmKeyword: string;
+    emmRiskFactor?: string;
+    count: number;
+}
+
+export interface EmmEntity {
+    name: string;
+}
+
+export interface Lead {
+    // TODO: Handle case where assignee can be multiple
+    assignee: number;
```

Comment:
any reason why multiple assignees cannot be assigned right now ?

Reply:
It is the requirement to set assignee as single select, but in old type I found it as number | number[].
Lets leave it as is for now, and fix it if we run into issues later.